### DISCUSSION TITLE
Will AdminLTE v4.0.0 drop jQuery as a dependency?

### DISCUSSION THREAD

**Discussion by dfsmania** [ColorlibHQ/AdminLTE#3732] [Q&A] [answered]
As a maintainer of a project related to this great template, I want to know if **AdminLTE v4** will drop `jQuery` as a dependency, so I can follow the same line on the project I maintain. Also, what you have planned for the plugins listed [here](https://adminlte.io/docs/3.1//dependencies.html) that depends on `JQuery`. Will they still be used for demo purposes?

**Comment by REJack** [comment, marked as answer]
Yep @Shidersz, all jquery based plugins will be removed/replaced. We will remove/replace also all selfmade jquery codes.
AdminLTE v4 will be complete jQuery free like Bootstrap 5 is 😄.

**Reply by dfsmania** [reply]
Thanks @REJack , I will take this into consideration to start migrating `JQuery` code of [Laravel-AdminLTE](https://github.com/jeroennoten/Laravel-AdminLTE) to pure `Javascript`.
